numba/cuda/tests/cudapy/test_array.py

```diff
@@ -61,5 +62,127 @@ def test_auto_device_const(self):
         d, _ = cuda.devicearray.auto_device(2)
         self.assertTrue(np.all(d.copy_to_host() == np.array(2)))
 
+    def _test_device_array_like_same(self, d_a):
+        """
+        Tests of device_array_like where shape, strides, dtype, and flags should
```

Comment:
think this should read "contiguity flags"? Not all flags are checked?

Reply:
Device arrays only have contiguity flags:

```
In [4]: arr = cuda.device_array_like(np.arange(10))                                                                                                          

In [5]: arr.flags                                                                                                                                            
Out[5]: {'F_CONTIGUOUS': True, 'C_CONTIGUOUS': True}
```

so I think it's accurate to suggest that all flags are checked.

Reply:
ah ok, thought there were more than that, consider this resolved!